.github/workflows/buildingRelease.yml

```diff
@@ -8,13 +8,47 @@ jobs:
       uses: actions/checkout@v2
       with:
         submodules: 'true'
+        
     - name: Download Subnautica files
       run: ./.github/depotDownloader/QXCTF.exe
+      
     - name: Setup Nuget.exe
       uses: nuget/setup-nuget@v1
+      
     - name: Restore packages
       run: nuget restore Nitrox.sln
+      
     - name: Setup MSBuild
       uses: microsoft/setup-msbuild@v1.0.2
+      
     - name: Build with MSBuild
-      run: msbuild Nitrox.sln -p:Configuration=Release
\ No newline at end of file
+      run: msbuild Nitrox.sln -p:Configuration=Release
+
+  # test:
+    # runs-on: windows-latest
+    # steps:
+    # - name: Checking out repository
+      # uses: actions/checkout@v2
+      # with:
+        # submodules: 'true'
+
+    # - name: Download Subnautica files
+      # run: ./.github/depotDownloader/QXCTF.exe
+
+    # - name: Setup Nuget.exe
+      # uses: nuget/setup-nuget@v1
+
+    # - name: Restore packages
+      # run: nuget restore Nitrox.sln
+
+    # - name: Setup MSBuild
+      # uses: microsoft/setup-msbuild@v1.0.2
+
+    # - name: Setup VSTest and add to PATH
+      # uses: darenm/Setup-VSTest@v1
+
+    # - name: Build Debug with MSBuild
+      # run: msbuild Nitrox.sln -p:Configuration=Debug
+
+    # - name: Run VSTest
+      # run: vstest.console.exe ./NitroxTest/bin/Debug/NitroxTest.dll
```

Comment:
Is this supposed to be commented out?

Reply:
Yeah, like written in the pr description GitHub is failing currently because the depot downloader isn't downloading resources.asset. So I comment it out until this is changed.